deploy/compose/.env.example

```
@@ -2,8 +2,10 @@ POSTGRES_USER=dhadgar
 POSTGRES_PASSWORD=dhadgar
 POSTGRES_DB=dhadgar_platform
 
-RABBITMQ_DEFAULT_USER=dhadgar
-RABBITMQ_DEFAULT_PASS=dhadgar
+# Used by both the RabbitMQ broker (RABBITMQ_DEFAULT_USER/PASS) and the
+# service clients (RabbitMq__Username/RabbitMq__Password) so they stay in sync.
+RABBITMQ_USERNAME=dhadgar
+RABBITMQ_PASSWORD=dhadgar
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_PASSWORD key should go before the RABBITMQ_USERNAME key

(UnorderedKey)